CHANGELOG.md

```diff
@@ -0,0 +1,1163 @@
+# Changelog
+
+## Qtile x.xx.x, released XXXX-XX-XX
+
+### Breaking changes
+
+- this release drops support for python 3.9
+- deleted the (very old) `libqtile/command_*` deprecation wrappers
+- SIGUSR2 no longer restarts qtile, instead it dumps stack traces
+- `lazy.<object>.when(when_floating=X)` now behaves differently: the lazy call will be executed independently of the current window's float state by default, and can be limited to when it is floating or tiled by passing `when_floating` as True or False respectively.
+
+### Features
+
+- automatically lift types to their annotated type when specified via the `qtile cmd-obj` command line
+- Add `Plasma` layout. The original layout (https://github.com/numirias/qtile-plasma) appears to be unmaintained so we have added this to the main codebase.
+- Add ability to specify muted and unmuted formats for `Volume` and `PulseVolume` widgets.
+
```

Comment:
Can we make this section less visible than the actual latest release on the HTML output somehow? E.g. commenting it out or using a spoiler, or anything else really

Reply:
Used a `details` HTML tag: https://pawamoy.github.io/qtile/manual/changelog/ :slightly_smiling_face:

Reply:
Pushed again to still include the `Qtile xxx` title (to fix the ToC)

Reply:
niceee